lib/c-api/src/wasm_c_api/types/frame.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Debug)]
```

Comment:
For types like frame and byte we should reexport them all at `types/mod.rs` or put them all in the same file I think

Reply:
Good to me.